components/script/dom/globalscope.rs

```diff
@@ -731,6 +731,13 @@ impl FileListener {
     }
 }
 
+fn now() -> u64 {
```

Comment:
I think it would be useful to indicate the units in the function name so something like `get_timestamp_in_ms`.